app/controllers/assignments_controller.rb

```diff
@@ -49,6 +52,13 @@ def destroy
 
   private
 
+  def student_identifier_types
+    @student_identifier_types ||= @organization.student_identifier_types.map do |student_identifier|
```

Comment:
WDYT about using `select` here? So that we only load what we need.

```ruby
@student_identifier_types ||= @organization.student_identifier_types.select(:name, :id).map do |student_identifier|
  [student_identifier.name, student_identifier.id]
end
```